ush/forecast_postdet.sh

```diff
@@ -60,6 +60,18 @@ FV3_postdet() {
           break
         fi
       done
+      # Replace fv_tracer with aeroanl_fv_tracer restart files from current cycle (if found)
+      local nn
+      for (( nn = 1; nn <= ntiles; nn++ )); do
+        if [[ -f "${COMOUT_ATMOS_RESTART}/${restart_date:0:8}.${restart_date:8:2}0000.aeroanl_fv_tracer.res.tile${nn}.nc" ]]; then
+          rm -f "${DATA}/INPUT/fv_tracer.res.tile${nn}.nc"
+          ${NCP} "${COMOUT_ATMOS_RESTART}/${restart_date:0:8}.${restart_date:8:2}0000.aeroanl_fv_tracer.res.tile${nn}.nc" \
+                 "${DATA}/INPUT/fv_tracer.res.tile${nn}.nc"
+        else
+          echo "WARNING: 'aeroanl_fv_tracer.res.tile1.nc' not found in '${COMOUT_ATMOS_RESTART}', using 'fv_tracer.res.tile1.nc'"
```

Comment:
```suggestion
          echo "WARNING: 'aeroanl_fv_tracer.res.tile${nn}.nc' not found in '${COMOUT_ATMOS_RESTART}', using 'fv_tracer.res.tile${nn}.nc'"
```
Should the check be on `tile1`?  What happens if `aeroanl_fv_tracer.res.tile1.nc` is present, but tile2.nc is not?  The logic here will `break` out, while tile 1 overwritten, but tile 2+ not.
Thoughts?

Reply:
I don't disagree, this is consistent with how it is used for the land analysis, see line 59. 

We should probably do something like the following pseudo-code, but I'm not immediately sure the best way to do this in shell:
```
use_anl_aero="YES"
for tile in tiles:
  if fv_tracer.tile${tile}.nc does not exist:
    use_anl_aero="NO"
    break
if use_anl_aero=="YES":
for tile in tiles: 
  rm
  cp 
```

Reply:
How about the following to replace lines 63-74.
```bash
      # Replace fv_tracer with aeroanl_fv_tracer restart files from current cycle (if found)
      local nn
      local use_anl_aero="YES"
      for (( nn = 1; nn <= ntiles; nn++ )); do
          test_tracer_file="${COMOUT_ATMOS_RESTART}/${restart_date:0:8}.${restart_date:8:2}0000.aeroanl_fv_tracer.res.tile${nn}.nc"
          if [[ ! -f  "${test_tracer_file}" ]]; then
              use_anl_aero="NO"
              echo "WARNING: File ${test_tracer_file} does not exist, will not replace any files"
          fi
      done
      if [[ use_anl_aero == "YES" ]]; then
        for (( nn = 1; nn <= ntiles; nn++ )); do
          rm -f "${DATA}/INPUT/fv_tracer.res.tile${nn}.nc"
          ${NCP} "${COMOUT_ATMOS_RESTART}/${restart_date:0:8}.${restart_date:8:2}0000.aeroanl_fv_tracer.res.tile${nn}.nc" \
                 "${DATA}/INPUT/fv_tracer.res.tile${nn}.nc"
        done
      fi
```

Reply:
Works for me, I made this change, except a minor modification to the warning, and a 'break' once the variable is set to NO